cmd/vic-machine/delete/delete.go

```diff
@@ -114,13 +116,36 @@ func (d *Uninstall) Run(cli *cli.Context) error {
 	// SetOutput to io.MultiWriter so that we can log to stdout and a file
 	log.SetOutput(io.MultiWriter(os.Stdout, f))
 
-	log.Infof("### Deleting VCH ####")
+	log.Infof("### Removing VCH ####")
 
 	validator := validate.NewValidator()
-	rp, err := validator.GetResourcePool(d.Data)
+	defer func() {
+		if validator.Session != nil {
+			validator.Session.Client.Logout(validator.Context)
+		}
+	}()
+
+	vch, err := validator.GetVCH(d.Data)
+	if err != nil {
+		err = errors.Errorf("%s. Exiting...", err)
```

Comment:
Unless we have  a structured method for marking whether an error is fatal or not, adding "Exiting..." onto the end makes it harder to have clean user focused output.
Consider adding log.Debug messages along with the pass through error check that explain whether the error is fatal or not